math-plugin/src/test/java/com/github/underscore/math/MathTest.java

```diff
@@ -279,84 +279,84 @@ public void chain() {
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).initial();
         $.chain(new String[] {""}).initial(1);
         $.chain(new String[] {""}).last();
         $.chain(new String[] {""}).last(1);
         $.chain(new String[] {""}).lastOrNull();
         $.chain(new String[] {""}).lastOrNull(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).rest();
         $.chain(new String[] {""}).rest(1);
         $.chain(new String[] {""}).compact();
         $.chain(new String[] {""}).compact("1");
         $.chain(new String[] {""}).flatten();
-        $.chain(new Integer[] {0}).map(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).map(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new String[] {""}).filter(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).filterIndexed(new PredicateIndexed<String>() {
-            public boolean apply(int index, String str) { return true; } });
+            public boolean test(int index, String str) { return true; } });
         $.chain(new String[] {""}).reject(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).rejectIndexed(new PredicateIndexed<String>() {
-            public boolean apply(int index, String str) { return true; } });
+            public boolean test(int index, String str) { return true; } });
         $.chain(new String[] {""}).filterFalse(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).reduce(new FunctionAccum<String, String>() {
             public String apply(String accum, String str) { return null; } }, "");
         $.chain(new String[] {""}).reduceRight(new FunctionAccum<String, String>() {
             public String apply(String accum, String str) { return null; } }, "");
         $.chain(new String[] {""}).find(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).findLast(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new Integer[] {0}).max();
-        $.chain(new Integer[] {0}).max(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).max(new Function<Integer, Integer>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.max\(Function\) ignored, is this OK in com\.github\.underscore\.math\.MathTest\.chain\(\)